_Week9/Group5/README.md

```diff
@@ -0,0 +1,43 @@
+# Week 9 - Group 5
+# Date: November 6, 2022
+
+## Overview
+We simulate data for our ECON 490 project studying DC charter schools. We use the DC system of admission by random lottery to charters as an instrument to estimate the impact of attending a charter of student test scores.
+
+## estimation.do
+This is our r-class program simulation. We generate 5,000 students who apply to the My DC Lottery for PK3. Each child has an innate ability and a designated local public school. The child applies to exactly one charter, which he/she chooses at random from amongst all charters with equal probability. Each school, public and charter, is randomly assigned a quality score. To reflect distributional differences, we draw public quality scores from a normal distribution with mean 100 and standard deviation 15, while charter scores are drawn from a normal distribution with the same standard deviation but mean of 105. We define the child's hypothetical test score attending a school as a Cobb-Douglas education production function of the child's innate ability and school quality. For schools where applicants exceeds capacity, a random lottery allocates seats. An admitted child chooses to attend the charter school if his/her test scores would be higher there than at the local public school. Children who lose the lottery and those whose test scores would be higher at their public schools choose to attend public schools.
+
+We define the true average treatment effect (ATE) as the effect of attending charters on test scores across all 5,000 children in our sample. The true ATE is simply the mean of the individual treatment effects (ITE) across all children (i.e., the difference between each child's test score attending charter and his/her test score attending public). It is mathematically equivalent to say the true ATE = mean(ts_charter) - mean(ts_public). We estimate the biased ATE using an IV regression of test scores on a dummy variable for attending a charter, where admission to a charter serves as an instrument. The regression coefficient represents the ATE across all students *who actually attend charters*, i.e., students who were admitted *and* who choose to attend (because their test scores would be higher attending charters than public).
+
+Why does the biased ATE differ from the true ATE? The bias is introduced by students who are admitted to charters but choose not to attend, as they would have lower test scores attending charters. As a result, the biased ATE is an overestimate of the true ATE. The latter reflects the impact of charter attendance across *all students*, whereas the biased ATE reflects the impact on the *compliers amongst the treated*, i.e., the admitted who *choose* to attend charters.
+
```

Comment:
I suggest we clarify the difference between the biased ATE and the true ATE here. We state that the regression coefficient of the biased ATE represents the ATE across all students who actually attend charters, i.e., students who were admitted and then choose to attend. We also say that the true ATE reflects the impact of charter attendance across *all students*. I think this might be confusing given that our treatment is attending a charter school. Let's clarify who the "all students" are in this case: all those who were admitted, regardless of whether they choose to attend?

Reply:
"The bias is introduced by students who are admitted to charters but choose not to attend, as they would have lower test scores attending charters." I'd like to suggest a small edit here. There are many reasons why a student may choose not to attend a charter school in real life, but in our study we assume that potential scores are the only reason. We should state this assumption explicitly here again, or else we will inevitably receive questions about this.

Reply:
Regarding your first comment, I clarified the language some more. Note "all students" here refers to all lottery entrants, not all admits. Please update as needed if you think it needs more clarity.

You make a good point in second comment. I added that bit to the file.